app/uk/gov/hmrc/gform/gform/SectionRenderingService.scala

```diff
@@ -2031,6 +2032,13 @@ class SectionRenderingService(
 
     val isPageHeading = ei.formLevelHeading
 
+    val singleton = ei.singleton
+    val page = singleton.page
+
+    val maybeDisplayWidth: Option[LayoutDisplayWidth] = None
```

Comment:
Isn't this `None` actually a missing implementation?

Reply:
Sorry, this is a missing implementation. Fixed.